### DISCUSSION TITLE
MIT 6.007 Signals and Systems - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#151] [Announcements]
# MIT 6.007 Signals and Systems - CS自学指南

CS自学指南

https://csdiy.wiki/%E7%94%B5%E5%AD%90%E5%9F%BA%E7%A1%80/Signals_and_Systems_AVO/

**Comment by Tonysssss** [comment]
计划重新刷ee的课，从祖师爷的ss开始刷起，后面会学dsp，dip 。ee的课任务量大，作业难度比较高，有想一起刷的朋友＋v 676503549

**Comment by scatyf3** [comment]
请问国内本科的“数字逻辑”课程是对应这篇指南的“电子基础”吗，大二小萌新想找一个对应的国外课学习，如果有大佬解答的话感激不尽！

**Reply by SeeingDragon** [reply]
请问你有找到吗？我也是想找数字逻辑的课

**Reply by ScienceVoyager** [reply]
EE16A&B不确定，但是EE120和6.007都是信号与系统，算是电子信息类的专业课，计算机专业一般不学，数字逻辑更类似于数字电子技术基础，可能有些院校在教学时选择深度和内容不及数字电子技术基础。

**Reply by taylover2016** [reply]
数字逻辑应该是讲的数电里的与或非以及异或之类的基础运算逻辑，可以看一下数字电路的相关课程。英文课程这边可以看一下伯克利的EECS151，B站有2020年的高清版。

**Reply by 124-lgtm-92** [reply]
可以考虑看任爱峰，B站有

**Comment by Tonysssss** [comment]
抱歉现在才看到这封邮件。


我没接触过数字逻辑，我感觉应该对应的是数字电路,电子基础的课基本上是信号方向的，不太一样的。
如果想要深入学习可以参考其他课程。










------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PKUFlyingPig/cs-self-learning"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月1日(星期四) 上午10:42
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [PKUFlyingPig/cs-self-learning] MIT 6.007 Signals and Systems - CS自学指南 (Discussion #151)





 
请问国内本科的“数字逻辑”课程是对应这篇指南的“电子基础”吗，大二小萌新想找一个对应的国外课学习，如果有大佬解答的话感激不尽！

**Comment by zhangeecs** [comment]
课程视频提供的B站合集有点糊，我这发现一个画质好点的，大家有需要自取：【画质增强版祖师爷奥本海姆《信号与系统》合集（ps:可能是全网画质最清晰的版本】 https://www.bilibili.com/video/BV1SB4y1C7yx

**Comment by lotefish** [comment]
要是有eediy就好力

**Reply by sist080900** [reply]
同求

**Reply by Eric30615** [reply]
同求

**Reply by js0ny** [reply]
同求

**Reply by zhuanghai1** [reply]
同求

**Reply by t1nD1** [reply]
同求

**Comment by m1a0cX** [comment]
有没有matlab上手的教程呀

**Reply by Sword4177** [reply]
你去Matlab的官网他们有官方的教学视频，我就是跟着那个学的
